src/Github/JwageChangelogGenerator.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\AutomaticReleases\Github;
+
+use ChangelogGenerator\ChangelogConfig;
+use ChangelogGenerator\ChangelogGenerator as BaseJwageChangelogGenerator;
+use ChangelogGenerator\IssueClient;
+use ChangelogGenerator\IssueFactory;
+use ChangelogGenerator\IssueFetcher;
+use ChangelogGenerator\IssueGrouper;
+use ChangelogGenerator\IssueRepository;
+use Doctrine\AutomaticReleases\Git\Value\SemVerVersion;
+use Doctrine\AutomaticReleases\Github\Value\RepositoryName;
+use GuzzleHttp\Client;
+use Symfony\Component\Console\Output\BufferedOutput;
+
+final class JwageChangelogGenerator implements ChangelogGenerator
+{
+    /** @var BaseJwageChangelogGenerator */
+    private $changelogGenerator;
+
+    public function __construct(BaseJwageChangelogGenerator $changelogGenerator)
+    {
+        $this->changelogGenerator = $changelogGenerator;
+    }
+
+    public static function create() : self
+    {
+        $guzzle          = new Client();
+        $issueClient     = new IssueClient($guzzle);
```

Comment:
Note: we already have `php-http` installed: if you can use a PSR-18 client, this is gonna work with the pre-installed one